### DISCUSSION TITLE
news/20231108/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#271] [Website Discussions]
# news/20231108/

...and that's a wrap!

https://fcp.cafe/news/20231108/

**Comment by florian-duffe** [comment]
Dear Chris. Thank you for your summit-summaries. With your enthusiastic writing it comes pretty much alive and I'm really thinking how to make it possible to attend next year! Thanks a lot!

**Comment by FilmYak** [comment]
Was great seeing you again, Chris.  And the rest of the FCP community there too, of course!  It was a very energizing and needed gathering. Very happy so many quality folks were able to attend.

**Reply by FilmYak** [reply]
Ps - since I’m too lazy to change my display name at this exact moment, this is Mike Yanovich.

**Comment by lenquist0** [comment]
Thanks for the update!
